### PR TITLE
Revise compound decision points documentation

### LINKED ISSUE #375: Update Compound Decision Points on Staging Site
In docs/topics/compound_decision_points.md, complete To Do tasks.

Include note that this is a compound decision point, therefore it is a notational convenience. 

Revise page.

Link: https://certcc.github.io/SSVC-staging/topics/compound_decision_points/#utility

### LINKED ISSUE #383: Update text of *Utility*
Should use proper decision point names, right? 

In https://github.com/CERTCC/SSVC/pull/370/files/083057d3d22b89f5309d9685428981d3f26a9713#diff-1aa025f17397e8e205d3bd93b041e65477a806878e53a8f9e71a08454339d1ab 

So "Value Density" instead of "value". 
Suggest being a little more explicit
"Answer to *Automatable* is *No* and answer to *Value Density* is *Diffuse* " or something like that? Is there a reason for the way this is written more tersely / informally?

_Originally posted by @j--- in https://github.com/CERTCC/SSVC/pull/370#discussion_r1387104995_

### PR DESCRIPTION
**Pull request by ahouseholder**
- resolves #375 
- resolves #383 

We had compound decision points showing up in both the Topics and Reference section. This consolidates all that into the Reference section.

It also updates the text of the python objects and generated JSON to more accurately reflect the proper decision point names.

### CODE REVIEW

File: docs/reference/decision_points/utility.md [hunk 1]
```diff
@@ -7,6 +7,42 @@
     Utility is a combination of [Automatable](./automatable.md) and
     [Value Density](./value_density.md)
 
+
+This is a compound decision point, therefore it is a notational convenience.
+
+*Utility* estimates an adversary's benefit compared to their effort based on the assumption that they can exploit the vulnerability.
+*Utility* is independent from the state of [*Exploitation*](exploitation.md), which measures whether a set of adversaries have ready access to exploit code or are in fact exploiting the vulnerability.
+
+*Utility* estimates an adversary's benefit compared to their effort based on the assumption that they can exploit the vulnerability.
```
Comment (j---) [inline docs/reference/decision_points/utility.md:16]: Is this intentionally a repetition of line 13?

File: docs/reference/decision_points/utility.md [hunk 2]
```diff
@@ -7,6 +7,42 @@
...
+*Utility* is independent from the state of [*Exploitation*](exploitation.md), which measures whether a set of adversaries have ready access to exploit code or are in fact exploiting the vulnerability.
```
Comment (j---) [inline docs/reference/decision_points/utility.md:17]: Is this intentionally a repetition of line 14?

**Review by j--- — CHANGES_REQUESTED** [review]
Minor questions about lines in docs/reference/decision_points/utility.md. Once those are answered, can merge, everything else looks good.

File: docs/reference/decision_points/utility.md (hunk 1 above)
Comment (ahouseholder) [reply docs/reference/decision_points/utility.md:16]: Oof. Bad file merge on my side. I'll fix this and the next one in an update to the branch next week.

File: docs/reference/decision_points/utility.md (hunk 2 above)
Comment (ahouseholder) [reply docs/reference/decision_points/utility.md:17]: See previous comment https://github.com/CERTCC/SSVC/pull/428#discussion_r1476743127 above

Comment (ahouseholder) [reply docs/reference/decision_points/utility.md]: fixed in 785f928

File: docs/reference/decision_points/utility.md (hunk 1 above)
Comment (ahouseholder) [reply docs/reference/decision_points/utility.md]: Fixed in 785f928

**Review by j--- — APPROVED** [review]

**Pull request merged**